src/components/sidebar/sidebar.css

```
@@ -12,6 +12,8 @@
 }
 
 .sidebar-contents-wrapper {
+  /* Allow absolute positioning that is independent of the scrolling. */
```

super nit: I'd use "dependent" instead of "independent" 
Adding the suggestion so that you can just merge it and merge the PR if you want to.
```suggestion
  /* Allow absolute positioning that is dependent of the scrolling. */
```